src/mdbExtensionController.ts

```diff
@@ -89,14 +115,32 @@ export default class MDBExtensionController implements vscode.Disposable {
     let mdbConnectionString;
     if (this._connectionController) {
       const activeConnectionConfig = this._connectionController.getActiveConnectionConfig();
-      mdbConnectionString = activeConnectionConfig ? activeConnectionConfig.driverUrl : '';
+      mdbConnectionString = activeConnectionConfig
+        ? activeConnectionConfig.driverUrl
+        : '';
     }
-    const mongoDBShell = vscode.window.createTerminal({ name: 'MongoDB Shell', env: { MDB_CONNECTION_STRING: mdbConnectionString } });
+    const mongoDBShell = vscode.window.createTerminal({
+      name: 'MongoDB Shell',
+      env: { MDB_CONNECTION_STRING: mdbConnectionString }
+    });
     const shellCommand = vscode.workspace.getConfiguration('mdb').get('shell');
-    mongoDBShell.sendText(`${shellCommand} $MDB_CONNECTION_STRING; unset MDB_CONNECTION_STRING`);
+    mongoDBShell.sendText(
+      `${shellCommand} $MDB_CONNECTION_STRING; unset MDB_CONNECTION_STRING`
+    );
     mongoDBShell.show();
   }
 
+  public createPlayground(): void {
```

Comment:
The auto prettifier changed lots of lines here. So this is the part that adds a new command for playground creation.